riff-raff/app/views/preview/yaml/showTasks.scala.html

```diff
@@ -13,18 +14,20 @@
     @b3.hiddens(
         "project" -> form("project").value.get,
         "build" -> form("build").value.get,
-        "stage" -> form("stage").value.get
+        "stage" -> form("stage").value.get,
+        "updateStrategy" -> form("updateStrategy").value.get
     )
     @defining(form("totalKeyCount").value){ maybeCount =>
       @maybeCount match {
           case Some(count) => { @b3.hidden("totalKeyCount", count) }
           case None => {
               <div class="alert alert-info">
                   This preview shows only the subset of deployment tasks that would be executed. To see all tasks, <a href="@routes.PreviewController.preview(
-                      form("project").value.get,
-                      form("build").value.get,
-                      form("stage").value.get,
-                      None
+                      form.value.map(_.project).get,
+                      form.value.map(_.build).get,
+                      form.value.map(_.stage).get,
```

Comment:
Should we use this syntax across the entire file?

Reply:
I thought I had - have just done the hiddens at the top if that was the one you were referencing?